### PR TITLE
ansible: changed heading to include subcommand specified

### PR DESCRIPTION
**Pull request by ireaneus**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/ansible.md [hunk 1]
```diff
@@ -1,10 +1,7 @@
 # ansible
 
-> Ansible installation can be found here.
-> [docs.ansible]<https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html>.
-
 > The following commands are examples of required steps to ensure ansible is installed properly.
-> Ansible may require a separate user account with sudo privs on each node to run system commands.
+> docs.ansible: <https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html>.
```
Comment (zdroid) [inline pages/common/ansible.md]:
```suggestion
> More information: <https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html>.
```

This should fix the issue.

**Comment by zdroid** [comment]
Please don't close the PR, if you wish to change the page.

However, this definitely needs to be reworked, since this is not our page format. It should be only one command per example, and those manpage-like lists of subcommands should be removed.

File: pages/common/ansible.md (hunk 1 above)
Comment (ireaneus) [reply pages/common/ansible.md]: I realize this cheatsheet modification isn't meant for tldr.  I am creating a repo designed for opensource cheatsheets that are not 1liners as it is formatted in tldr.  Closing Pull request.

**Comment by ireaneus** [comment]
Closing pull request, modified cheatsheet is not designed for tldr format

**Comment by sbrl** [comment]
Perhaps next time you could leave a helpful message instead of directly re-opening the PR, @zdroid? Here's one that I like to use:

```markdown
Hey - thanks for the new page! Looks like you're having some issues with the linter. We've got the style we prefer documented over [here](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#markdown-format), and a few good examples are the pages for `tar`, `eyeD3`, and `ln`. If you need any assistance, just ask :smiley_cat:
```

@ireaneus: No problem. Do let us know how you get on! We may be able to add your project to the [similar projects list](https://github.com/tldr-pages/tldr/#similar-projects) :D

**Comment by zdroid** [comment]
@sbrl I did try that, albeit in a less polite manner. :D 

> Please don't close the PR, if you wish to change the page.
>
> However, this definitely needs to be reworked, since this is not our page format. It should be only > one command per example, and those manpage-like lists of subcommands should be removed.

**Comment by sbrl** [comment]
Lol, didn't see that.

You might want to get into the pre-saved replies feature of GitHub. Since similar kinds of situations pop up semi-regularly, you can put lots of effort into a really nice message to send to people once, and recall them at speed. More information: https://help.github.com/en/github/writing-on-github/working-with-saved-replies

Here's a few of mine:

```
Thanks for the new page, @name! I've left some comments below for you to review.
```

```
Thanks for requesting this page! Do you want to have a go yourself?

Pages are best created by people who understand the command they are about.

If you do, take a quick read of [CONTRIBUTING.md](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md), which guides you through the process of creating a new page here. Then, just simply send a pull request to this repository - and if you need any assistance, just ask :-)
```

```
Hey - thanks for the new page! Looks like you're having some issues with the linter. We've got the style we prefer documented over [here](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#markdown-format), and a few good examples are the pages for `tar`, `eyeD3`, and `ln`. If you need any assistance, just ask :smiley_cat:
```

**Comment by zdroid** [comment]
Thanks for reminding me about that feature. I'll definitely add some saved replies.

**Pull request closed without merging**